NodeTypes/Document/Chapter/Chapter.fusion

```diff
@@ -23,6 +23,7 @@ prototype(Neos.Demo:Document.Chapter) < prototype(Neos.Fusion:Component) {
             block = false
         }
     }
+
     renderer = Neos.Demo:DocumentRenderer {
```

Comment:
Any ideas how I can extract the `@apply` from here?

Reply:
hmm maybe its too nested? @mficzel

Reply:
Since the book is removed anyways just rebase and this is gone anyways